### DISCUSSION TITLE
Datagrid - column with select checkbox and select all column header

### DISCUSSION THREAD

**Discussion by hahyes** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2739] [Q&A]
Hi there,
I followed known to me way to add to Datagrid column to select datagriditems
```
            <DataGrid.RowHeaderTemplate>
                <DataTemplate>
                    <Grid>
                        <CheckBox IsChecked="{Binding Path=IsSelected, Mode=TwoWay,
                     RelativeSource={RelativeSource FindAncestor,
                      AncestorType={x:Type DataGridRow}}}"/>
                    </Grid>
                </DataTemplate>
            </DataGrid.RowHeaderTemplate>
```

In default WPF it works as intended. In MaterialDesign it is not showing anything. Is there any way to achieve similar effect without too much code? Solution above was simple enough to implement, maybe I can do this with some fixes?

**Comment by ElieTaillard** [comment]
Converted to issue. Let's continue the discussion here https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2742
